pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/export/excel/ParticipantRecordData.java

```diff
@@ -62,11 +62,19 @@ public List<List<String>> processData(ParticipantWrapperResult participantData,
 
     }
 
-    private Collection<?> getJsonValue(Collection<?> nestedValue, ParticipantColumn column) {
+    private Collection<?> getJsonValue(Collection<?> nestedValue, ParticipantColumn column,
+                                       Alias alias) {
         if (nestedValue.isEmpty()) {
             return Collections.singletonList(StringUtils.EMPTY);
         }
-        String jsonString = nestedValue.stream().findFirst().get().toString();
+        String jsonString;
+        if (alias.isCollection()) {
+            jsonString = nestedValue.stream().filter(val ->
+                            column.getObject().equals(((Map<String, Object>) val).get("fieldTypeId")))
```

Comment:
can we please add `fieldTypeId` into a static string and not directly here, thank you